pandas/tests/extension/numpy_/conftest.py

```diff
@@ -0,0 +1,53 @@
+import pytest
+
+# A set of Base EA tests that are know to not work for
+# the object-dtype PandasArray holding nested data.
+skips = {
+    'BaseCastingTests.test_astype_str',
+    'BaseConstructorsTests.test_array_from_scalars',
+    # tuple isn't instance of np.object
+    'BaseGetitemTests.test_getitem_scalar',
+    # Can't pass tuples to _from_sequence
+    'BaseGetitemTests.test_take_series',
+    # np.array shape inference
+    'BaseInterfaceTests.test_array_interface',
+    # Can't construct expected.
+    'BaseMethodsTests.test_unique',
+    'BaseMethodsTests.test_combine_add',
+    'BaseMethodsTests.test_shift_fill_value',
+    'BaseMethodsTests.test_where_series',
+    'BaseMethodsTests.test_repeat',
+    # Can't hash ndarray[tuple]
+    'BaseMethodsTests.test_hash_pandas_object_works',
+    # Can't construct expected.
+    'BaseReshapingTests.test_merge',
+    'BaseReshapingTests.test_merge_on_extension_array',
+    'BaseReshapingTests.test_merge_on_extension_array_duplicates',
+
+    # ndarray setting
+    'BaseSetitemTests.test_setitem_scalar_series',
+    'BaseSetitemTests.test_setitem_sequence',
+    'BaseSetitemTests.test_setitem_sequence_mismatched_length_raises',
+    'BaseSetitemTests.test_setitem_sequence_broadcasts',
+    'BaseSetitemTests.test_setitem_sequence_broadcasts',
+    'BaseSetitemTests.test_setitem_loc_scalar_mixed',
+    'BaseSetitemTests.test_setitem_iloc_scalar_mixed',
+    'BaseSetitemTests.test_setitem_loc_scalar_multiple_homogoneous',
+    'BaseSetitemTests.test_setitem_iloc_scalar_multiple_homogoneous',
+    'BaseSetitemTests.test_setitem_mask_broadcast',
+    'BaseSetitemTests.test_setitem_scalar_key_sequence_raise',
+
+    'BaseParsingTests.test_EA_types',
+}
+
+
+def pytest_collection_modifyitems(config, items):
+    skip = pytest.mark.skip(reason="Skipping this because ...")
+    for item in items:
+        # TODO: See if pytest has a better way to resolve the *value*
```

Comment:
I'd be eager to hear of a better way to do this. Right now, we rely on the name being like `TestMethods::test_where_series[object-True]` do determine if we should skip. Not the cleanest. Ideally we would be able to resolve that to the actual value, but pytest may not have that yet.

Reply:
@simonjayhawkins any thoughts here

Reply:
@TomAugspurger why don't you just add a marker to specific tests? (and skip on that)

Reply:
I'm not aware of a way for a test marker to get access to the value of another fixture.

Reply:
> Ideally we would be able to resolve that to the actual value, but pytest may not have that yet.

IIUC, from a test or fixture you get the value supplied to the fixture by including that (used by) fixture in the function signature. seems to work...

https://github.com/simonjayhawkins/pandas/blob/6ebf2c52d8925663a9242d3c58a069000b2c3f06/pandas/tests/io/formats/conftest.py#L141-L146

that would mean changing all the tests individually though. so this may not be what your after.